src/observable/ConnectableObservable.ts

```diff
@@ -5,10 +5,10 @@ import {Subscription} from '../Subscription';
 
 export class ConnectableObservable<T> extends Observable<T> {
 
-  subject: Subject<T>;
-  subscription: Subscription;
+  protected _subject: Subject<T>;
```

Comment:
@kwonoj 

Ah, I felt there are some implicit customs which is start with `_` to mark a property as a private.
okay. I'll fix it